spec/valkyrie/persistence/fedora/metadata_adapter_spec.rb

```diff
@@ -41,10 +51,19 @@
 
       describe "#pair_path" do
         it "creates pairs until the first dash" do
-          expect(adapter.pair_path('abcdef-ghijkl')).to eq('ab/cd/ef')
+          id = 'abcdef-ghijkl'
+          if adapter.fedora_version == 6
+            expect(adapter.pair_path(id)).to eq('ab/cd/ef/-g')
+          else
+            expect(adapter.pair_path(id)).to eq('ab/cd/ef')
+          end
         end
         it "creates pairs until the first slash" do
-          expect(adapter.pair_path('admin_set/default')).to eq('ad/mi/n_/se/t')
+          if adapter.fedora_version == 6
+            expect(adapter.pair_path('adminset/default')).to eq('ad/mi/ns/et')
```

Comment:
How about using the same test case as non-fedora 6 for side-by-side comparison?
```suggestion
            expect(adapter.pair_path('admin_set/default')).to eq('ad/mi/n_/se')
```

Reply:
@cjcolvar All examples normalized to always use the same ID value regardless of Fedora version.